owasp-top10-2017-apps/a6/stegonography/README.md

```diff
@@ -75,21 +77,43 @@ By visiting `http://localhost:10006/login` we get to the following screen:
     <img src="images/login_page.png"/>
 </p>
 
-A quick `admin:admin` guess revealed that we can successfully log in to the application and get to the admin's control-panel, as shown by the image bellow:
+### 🔥
+
+A quick `admin:admin` guess revealed that we can successfully log in to the application and get to the admin's control-panel, as shown by the image below:
 
 <p align="center">
     <img src="images/admin_page.png"/>
 </p>
 
+### 👀
+
 #### Verbose session token gives away unnecessary information
 
-After logging in to the application, it's possible to see it sets a session token: `jwtSessionToken`. As shown by the following image:
+After logging in to the application, it's possible to see it sets a session token: `nodejsSessionToken`. As shown by the following image:
 
 <p align="center">
-    <img src="images/jwt_token.png"/>
+    <img src="images/token.png"/>
 </p>
 
-Even though using verbose tokens might not pose a straight forward security vulnerability, on certain apps it might give away the hint an attacker needs to compromise it's security. 🕵️‍
+### 🔥
+
+Having a look at the token's name, we get a strong indication that the app might be running NodeJS. By using `searchsploit` an attacker could find a malicious code to exploit this vulnerability.
```

Comment:
I think this phrase could be rewritten to something like:
"By using `searchsploit` an attacker could find a malicious code to exploit a NodeJs application vulnerability."
The way it currently is written makes it look like NodeJs is a vulnerability.

Reply:
Thanks @Takehime! I agree with you! I'm changing that sentence!